lib/streamlit/DeltaGenerator.py

```diff
@@ -1753,8 +1754,8 @@ def slider(
             The maximum permitted value.
             Defaults 100 if the value is an int, 1.0 otherwise.
         value : int/float or a tuple/list of int/float or None
-            The value of this widget when it first renders. In case the value
-            is passed as a tuple/list a range slider will be used.
+            The value of this widget when it first renders. A range slider
+            is used to support tuples/lists.
```

Comment:
This reads really strangely. The idea here is that if you pass a two-element tuple/list then the slider that shows up on the screen will be a "range slider". This new text doesn't really convey that.

Reply:
Fixing this in the next push.

Reply:
I've done a full rewrite on this one. Sorry, the last rev wasn't very good.